### PR TITLE
[doc] Unify data source switching of the docs during installation

### LINKED ISSUE #10566: [Feature][Doc] unified data source switching of the docs during installation
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

unified data source switching of the docs during installation

### Use case

above.

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

close #10566

### CODE REVIEW

File: docs/docs/en/guide/howto/general-setting.md [hunk 1]
```diff
@@ -28,3 +28,71 @@ click the button on the right of the [language](#language) button and then click
 you want to switch. All time related components will adjust their time zone according to the time zone setting you select.
 
 
+## Standalone Switching Metadata Database Configuration
+
+We here use MySQL as an example to illustrate how to configure an external database:
```
Comment (zhongjiajie) [inline docs/docs/en/guide/howto/general-setting.md]: Can we create an new file to keep those content? I will migrate how to use other database in "datasource center" in the ferther, currently they are in https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/datasource/introduction.html

Comment (SbloodyS) [reply docs/docs/en/guide/howto/general-setting.md]: Which path should we put into this new file?

Comment (zhongjiajie) [reply docs/docs/en/guide/howto/general-setting.md]: I think under the directory `HOWTO` and same level with `general-setting.md` is a great place

**Comment by zhongjiajie** [comment]
> ## Purpose of the pull request
> close #10566

Great, and thank for you bring this up

File: docs/docs/en/guide/howto/general-setting.md (hunk 1 above)
Comment (SbloodyS) [reply docs/docs/en/guide/howto/general-setting.md]: Done. PTAL @zhongjiajie

**Review by zhongjiajie — COMMENTED** [review]
I have two suggestion.

* we should add new page to siderbar, https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js. But we already `how to`  to FAQ sub sidebar. So I am not sure where is the best place to keep it. maybe we should add it next to https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js#L383
* I think we should combine existing datasource setting from https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/datasource/introduction.md#using-datasource-incompatible-to-apache-license-v2-license to `how-to/datasource-setting.md`

**Comment by SbloodyS** [comment]
> I have two suggestion.
> 
> * we should add new page to siderbar, https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js. But we already `how to`  to FAQ sub sidebar. So I am not sure where is the best place to keep it. maybe we should add it next to https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js#L383
> * I think we should combine existing datasource setting from https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/datasource/introduction.md#using-datasource-incompatible-to-apache-license-v2-license to `how-to/datasource-setting.md`

That's good. I'll add it.

**Comment by zhongjiajie** [comment]
FYI, one deadlink found https://github.com/apache/dolphinscheduler/runs/7176457415?check_suite_focus=true

**Comment by zhongjiajie** [comment]
Still have deadlink on your patch

**Comment by SbloodyS** [comment]
> Still have deadlink on your patch

I'm fixing it...

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**